internal/router/fieldinclude.go

```diff
@@ -58,6 +64,10 @@ type FieldGetFunc func(doc interface{}, id *ResolvedURL, path string, flags url.
 // after the initial prefix has been removed.
 type FieldPutFunc func(id *ResolvedURL, path string, val *json.RawMessage, updater *FieldUpdater, req *http.Request) error
 
+// CopyACLsFunc copies the ACL from then base entity id into all entities
+// with that base entity.
+type CopyACLsFunc func(id *ResolvedURL) error
```

Comment:
similarly:

// UpdateACLsFunc updates the ACL information for the given URL.
type UpdateACLsFunc func(*ResolvedURL) error

?